enterprise/payara.tooling/src/org/netbeans/modules/payara/tooling/data/JDKVersion.java

```diff
@@ -241,15 +257,15 @@ public String toString() {
 
     public static JDKVersion toValue(String version) {
         if (version != null && version.matches(VERSION_MATCHER)) {
-            return new JDKVersion(version, null);
+            return new JDKVersion(version, null, null);
         } else {
             return null;
         }
     }
 
-    public static JDKVersion toValue(String version, String vendor) {
+    public static JDKVersion toValue(String version, String vendor, String vm) {
```

Comment:
`JDKVersion ` is not used by other modules.

Reply:
@jGauravGupta well it was exposed to other modules, and is therefore caught by the sigtests. There is still a failure in JvmConfigReader. IMO the original signatures need adding back to match the old generated sigtests - that's what they're there for.  However, as it's friend-only exposure at the moment, there's an argument it can evolve.

Reply:
Keep the original method and let it call the new one.